src/vma/util/vma_stats.h

```diff
@@ -188,6 +189,10 @@ typedef struct {
 	uint32_t                     n_tx_ready_byte_count;
 	socket_counters_t            counters;
 	std::bitset<MC_TABLE_SIZE>   mc_grp_map;
+	ring_logic_t		     ring_alloc_logic_rx;
+	ring_logic_t		     ring_alloc_logic_tx;
```

Comment:
Any reason to print the ring logic in the statistics ? it always contains the value of VMA_RING_ALLOCATION_LOGIC_TX/VMA_RING_ALLOCATION_LOGIC_RX

Reply:
Not really, with the new feature, user can change the logic of the socket at any stage and it can be different than the default logic.